src/dialogs/more-info/components/lights/light-color-temp-picker.ts

```diff
@@ -111,7 +112,7 @@ class LightColorTempPicker extends LitElement {
 
     if (stateObj.state === "on") {
       this._ctPickerValue =
-        stateObj.attributes.color_mode === LightColorMode.COLOR_TEMP
+        lightSupportsColorMode(stateObj, LightColorMode.COLOR_TEMP)
```

Comment:
This change is not correct, just having the attribute does not indicate the current mode the light is in.

../Frenck